source/Outfit.cpp

```diff
@@ -187,18 +187,27 @@ namespace {
 void Outfit::Load(const DataNode &node)
 {
 	if(node.Size() >= 2)
-	{
-		name = node.Token(1);
-		pluralName = name + 's';
-	}
+		trueName = node.Token(1);
+
 	isDefined = true;
 
+	// Use bools to track whether the display and plural names are set as opposed to string.empty() to allow explicitly empty names.
+	bool displayNameSet = false;
```

Comment:
I don't agree, creating an outfit with an empty display name is poor user experience. What does allowing it let a content creator do that they could not do by seeing the display name to be "???” or "(unknown outfit)"?

If I saw some outfit for plunder or in my cargo that had no name, I would just think it was a bug with the game, not an intentional choice.